### ISSUE TITLE
E-tree: We're having some trouble with this experience. Try to close and reopen E-tree.

### ISSUE THREAD

**Issue by JoseErnestoOnGithub** [JoseErnestoOnGithub/E-tree-Errors-on-The-Day#1]
**Describe the bug**
E-tree fails to load with "We're having some trouble with this experience. Try to close and reopen E-tree.".

**To Reproduce**
1. Open E-tree
2. Show this error.

**Expected behavior**
E-tree failed to load.

**Screenshots**
![image](https://user-images.githubusercontent.com/115112109/212775486-3b547008-f249-4776-b37c-85657de0e299.png)

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro (activated with product key)
 - Browser: Microsoft Edge
 - Version: Unspecified

**Additional context**

**Comment by JoseErnestoOnGithub** [comment]
closin' because of not enough activity

**Comment by JoseErnestoOnGithub** [comment]
![Capture d’écran 2023-01-30 151311](https://user-images.githubusercontent.com/115112109/215617956-f0d41c83-76d9-4e84-b84a-f63155c00205.png)
Now it works again after creating a new forest.

**Comment by Computer-Tsu** [comment]
I started E-Tree on a Win 10 PC. When I setup a second computer, with Win 11, I logged in to my Microsoft account, synced Edge, and when opening E-Tree on this new computer, I get the error message "We're having some trouble with this experience. Try to close and reopen E-tree." The error has not cleared in over a week on the second computer while the first computer continues to open E-tree fine.
I do see the E-tree in the Win 11 Weather app and seem to be earning additional points there.
